content/chainguard/chainguard-images/pricing.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Chainguard Container Catalog Pricing"
+linktitle: "Catalog Pricing"
+type: "article"
+description: "Explains Chainguard Container catalog pricing, repository quotas, renaming, and sync, and helps you understand how to make the most of your subscription."
+date: 2025-08-19T08:49:31+00:00
+lastmod: 2025-08-19T08:49:31+00:00
+draft: false
+tags: ["Chainguard Containers", "pricing"]
+images: []
+menu:
+  docs:
+    parent: "chainguard-images"
+weight: 055
+toc: true
+---
+
+Catalog pricing is Chainguard's pricing model for container usage that replaces the manual ticket process for adding images to an organization. This enables you to self-service request individual images in the console to be added to your repository from the wider Chainguard catalog.
+
+The catalog pricing model is an option that requires you to be on the Catalog Pricing plan.
```

Comment:
Should "Catalog Pricing" be title case? If so, it should be consistent throughout (first sentence it's not title case, for instance).